plugin/frame_events.ml

```diff
@@ -147,7 +147,10 @@ let of_operand_info arch oi =
   | `mem_operand mo ->
     of_mem_operand arch mo oi.operand_usage oi.bit_length oi.value
   | `reg_operand ro ->
-    of_reg_operand arch ro oi.operand_usage oi.bit_length oi.value
+    let length = 
```

Comment:
this looks like a local patch to me. 
Why the bit_length on the first hand is `0`? Why we use `value * 8` to get the `bit_length` if it is zero. It looks like, that the problem is in the tracer, not in the trace. 

Moreover, this trace format can be used by different tracers, we have at least two tracers, at the time of writing, that uses this format. Are you sure that they both conforms with this subtle behavior?